src/chrome/content/rules/FreeRADIUS.org.xml

```diff
@@ -0,0 +1,22 @@
+<!--
+	Invalid Certificate:
+		- bugs.freeradius.org
+		- ftp.freeradius.org
+		- git.freeradius.org
+		- lists.freeradius.org
+		- mail.freeradius.org
+		- ns2.freeradius.org
+		- power.freeradius.org
+		- luajit.wiki.freeradius.org
+	Problematic HTTPS configuration:
```

Comment:
`Problematic HTTPS configuration` is not very informative. Please use `Broken certificate chain` instead or put those subdomains in the `Invalid certificate` section which in note that the chain is missing.